lib/ed25519-browser.js

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as ed25519 from '@stablelib/ed25519';
+import {randomBytes} from '@stablelib/random';
+
+export default {
+  ...ed25519,
+  async generateKeyPair() {
+    return ed25519.generateKeyPair({randomBytes: () => randomBytes(32)});
+  }
+};
```

Comment:
No need to create a new function every time we generate a key pair:

```suggestion
export default {
  ...ed25519,
  async generateKeyPair() {
    const seed = await randomBytes(32);
    return ed25519.generateKeyPairFromSeed(seed);
  }
};
```

Reply:
`@stablelib/ed25519` needs a function. I did create one that is re-usable that can be passed in here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/0fcf12662180890a3fdce649b03583a089b738fc